lib/main.js

```diff
@@ -1,11 +1,11 @@
 process.title = 'elm-review';
 
 if (!process.argv.includes('--no-color')) {
-  if (process.env['NO_COLOR'] === undefined) {
+  if (process.env['NO_COLOR'] === "1") {
```

Comment:
https://no-color.org/ says:

> Command-line software which adds ANSI color to its output by default should check for a `NO_COLOR` environment variable that, _when present and not an empty string **(regardless of its value)**_, prevents the addition of ANSI color.

The next paragraph I’ve always interpreted as an _example:_

> By adopting this standard, users that prefer to have plain, non-colored text output can export `NO_COLOR=1` to their shell’s environment and automatically disable color by default in all supported software.

As far as I know, `NO_COLOR=0` also turns off color. It’s only based on the presence, not the value.

Reply:
Gotcha, my bad. But doing `No_COLOR=""` should allow for colors (I tried this on the GitHub CLI, one of the projects they list in that file). I'll fix that.

Reply:
Fixed on master (I amended the history a bit to remove confusion)

Reply:
`NO_COLOR=""` is probably inconsistent between tools, likely due to the empty string being falsy in many languages.

In elm-watch and other programs I’ve made I just do `if ("NO_COLOR" in process.env)` because I interpreted the “spec” as if the `NO_COLOR` is set at all, regardless of value (including the empty string), it should mean “no colors”.

I don’t care about which way you go in elm-review though, just wanted to share this detail 😄